src/drivers/device/CMakeLists.txt

```diff
@@ -61,9 +61,4 @@ else()
 
 endif()
 
-px4_add_module(
-	MODULE drivers__device
-	SRCS ${SRCS}
-	DEPENDS
-		platforms__common
-	)
\ No newline at end of file
+px4_add_library(drivers_device ${SRCS})
```

Comment:
Needs newline

Reply:
Why do we need newlines at the end of a cmake file?